lib/infobox/node.js

```diff
@@ -248,11 +264,17 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
     attributeEntry(attributes, "Site", showSite(d, config))
     attributeEntry(attributes, "Uptime", showUptime(d))
     attributeEntry(attributes, "Teil des Netzes", showFirstseen(d))
+    attributeEntry(attributes, "Wlan Kanal 2.4 GHz", showWifiChannel(dictGet(d.nodeinfo, ["wireless", "chan2"])))
+    attributeEntry(attributes, "Wlan Kanal 5 GHz", showWifiChannel(dictGet(d.nodeinfo, ["wireless", "chan5"])))
     attributeEntry(attributes, "Systemlast", showLoad(d))
     attributeEntry(attributes, "Arbeitsspeicher", showRAM(d))
+    attributeEntry(attributes, "Airtime 2.4 GHz", showAirtime(2, dictGet(d.statistics, ["wireless", "airtime2"])))
+    attributeEntry(attributes, "Airtime 5 GHz", showAirtime(5, dictGet(d.statistics, ["wireless", "airtime5"])))
     attributeEntry(attributes, "IP Adressen", showIPs(d))
     attributeEntry(attributes, "Webseite", showPages(d))
     attributeEntry(attributes, "Gewähltes Gateway", dictGet(d.statistics, ["gateway"]))
+    attributeEntry(attributes, "Gewählter Nexthop", dictGet(d.statistics, ["nexthop"]))
+    attributeEntry(attributes, "Gewählter Nexthop", dictGet(d.statistics, ["gateway_nexthop"]))
```

Comment:
I'll come up with a function which takes care of displaying gateway and gateway_nexthop if it is unequal to gateway. I should probably display both values in one line then already.